minutes/2022-08-22.md

```diff
@@ -0,0 +1,95 @@
+# Meeting Minutes for August 22, 2022
+
+## Attendees
+
+@fviernau, @MarcelBochtler, @mnonnenmacher, @sschuberth, @tsteenbe
+
+## NOTICE file and copyright change
+
+- New files will be created with the new copyright header, see https://github.com/oss-review-toolkit/ort-governance/issues/6#issuecomment-1173522852.
+- Existing files should not be updated using the new copyright header. This will be done by Sebastian in one big change.
```

Comment:
Maybe use `@sschuberth` instead of `Sebastian`.

Reply:
I intentionally used the first names instead of the GitHub usernames, to be consistent with previous meeting minutes.
Also, these names are clarified here: https://github.com/oss-review-toolkit/ort-governance#minutes

Reply:
Ok, then if at all, let's change them on bulk some time afterwards.